### PR TITLE
Create event bot staking

### PR DESCRIPTION
**Pull request by dwalintukan**
Merge when this one is merged: https://github.com/bodhiproject/bodhi-graphql/pull/154

### CODE REVIEW

File: src/redux/Topic/saga.js [hunk 1]
```diff
@@ -6,6 +6,40 @@ import { request } from '../../network/httpRequest';
...
+      const options = {
+        method: 'POST',
+        body: JSON.stringify({
+          senderAddress,
+        }),
+        headers: { 'Content-Type': 'application/json' },
+      };
+
+      const result = yield call(request, Routes.eventEscrowAmount, options);
+      const eventEscrowAmount = satoshiToDecimal(result[0]);
```
Comment (liviavinci) [inline src/redux/Topic/saga.js]: why [0]? in order words, why is result an array?

File: src/scenes/CreateEvent/index.js [hunk 2]
```diff
@@ -120,23 +120,26 @@ class CreateEvent extends React.Component {
   static propTypes = {
     classes: PropTypes.object.isRequired,
     fields: PropTypes.array.isRequired,
-    open: PropTypes.bool.isRequired,
-    onClose: PropTypes.func.isRequired,
+    walletAddresses: PropTypes.array.isRequired,
+    setLastUsedAddress: PropTypes.func.isRequired,
```
Comment (liviavinci) [inline src/scenes/CreateEvent/index.js]: this is not used right?

File: src/redux/Topic/saga.js (hunk 1 above)
Comment (dwalintukan) [reply src/redux/Topic/saga.js]: The key is actually `"0"` which corresponds to the first return param.

File: src/scenes/CreateEvent/index.js (hunk 2 above)
Comment (dwalintukan) [reply src/scenes/CreateEvent/index.js]: yup you are right, removed

**Pull request merged**